server.js

```diff
@@ -41,54 +28,25 @@ const Movie = mongoose.model('Movie', {
 if (process.env.RESET_DATABASE) {
   console.log('resetting database!')
   const populateDatabase = async () => {
-    await Movie.deleteMany();
+    await Production.deleteMany();
   
     netflixData.forEach(item => {
-      const newMovie = new Movie(item);
-      newMovie.save();
+      const newProduction = new Production(item);
+      newProduction.save();
```

Comment:
Love this, nice to see that you have challenged yourself to include another route. Props to that.